source/get-started/pojo-qs.txt

```diff
@@ -0,0 +1,345 @@
+
+Inserting a POJO into MongoDB
+-----------------------------
+
+The codec registry automatically tries to create a POJO ``Codec`` for
+unknown classes. This allows you to use POJOs out of the box without
+any extra configuration.
+
+Before you can insert a POJO into MongoDB, create a
+``MongoCollection`` instance configured with the POJO class:
+
+.. code-block:: java
+
+   MongoCollection<Person> collection = database.getCollection("people", Person.class);
+
+Insert a Person Instance
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _javars-pojo-insertone:
+
+To insert a ``Person`` into the collection, use the collection's
+``insertOne()`` method:
+
+.. code-block:: java
+
+   Person ada = new Person("Ada Byron", 20, new Address("St James Square", "London", "W1"));
+   collection.insertOne(ada).subscribe(new OperationSubscriber<InsertOneResult>());
+
+Insert Multiple Person Instances
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To insert multiple ``Person`` instances, you can use the collection's
+``insertMany()`` method, which takes a list of ``Person`` instances as a
+parameter.
+
+The following example will add multiple ``Person`` instances into the
+collection:
+
+.. code-block:: java
+
+   List<Person> people = asList(
+       new Person("Charles Babbage", 45, new Address("5 Devonshire Street", "London", "W11")),
+       new Person("Alan Turing", 28, new Address("Bletchley Hall", "Bletchley Park", "MK12")),
+       new Person("Timothy Berners-Lee", 61, new Address("Colehill", "Wimborne", null))
+   );
+   collection.insertMany(people).subscribe(new OperationSubscriber<InsertManyResult>());
+
+Query the Collection
+--------------------
+
+To query the collection, you can use the ``find()`` method.
+
+The following example prints all the ``Person`` instances in the
+collection:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: java
+
+      collection.find().subscribe(new PrintToStringSubscriber<>());
+
+   .. output::
+      :language: none
+      :visible: false
+
+      Person{id='...', name='Ada Byron', age=20, address=Address{street='St James Square', city='London', zip='W1'}}
+      Person{id='...', name='Charles Babbage', age=45, address=Address{street='5 Devonshire Street', city='London', zip='W11'}}
+      Person{id='...', name='Alan Turing', age=28, address=Address{street='Bletchley Hall', city='Bletchley Park', zip='MK12'}}
+      Person{id='...', name='Timothy Berners-Lee', age=61, address=Address{street='Colehill', city='Wimborne', zip='null'}}
+
+Specify a Query Filter
+----------------------
+
+To query for ``Person`` instances that match certain conditions, pass a
+filter object to the ``find()`` method. To facilitate creating
+filter objects, the driver provides ``Filters`` helper methods.
+
+.. important::
+
+   When querying POJOs, you *must* query against the document field name
+   and not the POJO property name. By default, they are the same but it
+   is possible to change how POJO property names are mapped.
```

Comment:
S: I think the "but it is possible to change..." clause should have a comma before it
```suggestion
   and not the POJO property name. They are the same by default, but it
   is possible to change how POJO property names are mapped.
```